### ISSUE TITLE
Language setting not work when Windows display language is different from system language

### ISSUE THREAD

**Issue by AC19970** [mpvnet-player/mpv.net#692]
It seems that the interface language setting does not take into account the Windows display language. If the user's display language is different from the system setting, the software will only display according to the Windows system setting. And in this case, after manually setting it to a language other than English, the interface is still in English after restarting the software. I used the software on another device where the system language and the interface display language are the same and neither is English, and the language setting worked properly.

mpv.net version: 7.1.1.0

**Comment by stax76** [comment]
Sorry about the late reply.

Your explanation is difficult for me to understand.

What is the output if you run `Get-UICulture` in PowerShell?

There was a language related fix recently, it's not yet released, but you can try an auto build:

https://github.com/mpvnet-player/mpv.net/actions/runs/9947283193/artifacts/1703901208

**Comment by AC19970** [comment]
I'm sorry that my description wasn't accurate enough. I have a Windows 10 computer with English as the system language setting (system locale, the output of Get-UICulture and non-unicode language in Area setting) and I set the interface language (Windows Settings → Time and Language) to Chinese and Japanese, with English as fallback. In this case, the software does not get my interface language, and still only displays English when I manually set the language. When I ran it on another computer where all settings were in Chinese, the software's interface language was correctly displayed as Chinese without further settings. I tried the latest build but it didn't solve the issue.

**Comment by stax76** [comment]
So far I cannot reproduce it. I use this setup:

Desktop> Get-UICulture; Get-Culture

```
LCID             Name             DisplayName
----             ----             -----------
1033             en-US            English (United States)
1031             de-DE            German (Germany)
```

I tried changing the UI language to German, using `System` language in mpv.net showed German in mpv.net as expected, using Japanese from CLI in mpv.net showed Japanese in mpv.net as expected.

`Desktop> mpvnet --language=japanese`

So here it works as expected. At this point, I don't know what to do. Maybe somebody else reports the same problem. It's difficult to reproduce exactly for me as I don't understand Chinese and Japanese, so I cannot change the Windows UI language to Chinese or Japanese.

**Comment by AC19970** [comment]
I tried the method you mentioned and also tried using Locale Emulator to fake the system language, but the problem still exists. I will test it with other devices later.

**Comment by HayashiKaori** [comment]
I encountered the same issue, which might be caused by old configuration files affecting the language setting in version 7.1.1.0. 
I resolved it using the following method:

In the UI, right-click -> Settings -> Open Config Folder, open the following directory:
C:\Users\[your user name]\AppData\Roaming

Then delete the mpv and mpv.net folders. After that, reconfigure the language options.

You cannot reproduce this issue on another PC because that one does not have the old version's configuration.
